jobs/build/rhcos/Jenkinsfile

```diff
@@ -86,7 +86,8 @@ node {
                 currentBuild.displayName += "multi"
                 echo "triggering multi builds"
                 def jenkins_url = 'https://jenkins-rhcos.apps.ocp-virt.prod.psi.redhat.com'
-                commonlib.shell(script: "pip install -e ./art-tools/pyartcd && rm -rf ./artcd_working && mkdir -p ./artcd_working")
+                commonlib.shell(script: "pip install -e ./art-tools/pyartcd")
```

Comment:
do we need this, i think we're doing this in buildllib init?

Reply:
Saw we have this in other places too, e.g. [here](https://github.com/openshift-eng/aos-cd-jobs/blob/master/jobs/build/promote-assembly/Jenkinsfile#L120). Let's tackle this change in a separate PR